### PR TITLE
Feature italics abstract methods

### LINKED ISSUE #7346: Let pyreverse show abstract methods differently
### Current problem

Currently, there's no way to distinguish abstract methods.

### Desired solution

Abstract methods are shown in italic in UML diagrams.
I propose to get that idea and shows `@abstractmethod` also in italic.
CC @qequ

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by qequ**
- [x] Write a good description on what the PR does.
- [x] Create a news fragment with `towncrier create <IssueNumber>.<type>` which will be
  included in the changelog. `<type>` can be one of: new_check, removed_check, extension,
  false_positive, false_negative, bugfix, other, internal. If necessary you can write
  details or offer examples on how the new change is supposed to work.
- [x] If you used multiple emails or multiple names when contributing, add your mails
      and preferred name in ``script/.contributors_aliases.json``

## Type of Changes

|      | Type                   |
| --- | ---------------------- |
| ✓   | :sparkles: New feature |

## Description
Update Pyreverse's dot, plantuml and Mermaid.js printers to detect when class methods are abstract and show them with italic font. For the dot printer update the label to use html-like syntax.

Closes #7346

CC: @dgutson

### CODE REVIEW

File: pylint/pyreverse/dot_printer.py [hunk 1]
```diff
@@ -92,17 +92,22 @@ def _build_label_for_node(self, properties: NodeProperties) -> str:
 
         # Add class attributes
         attrs: list[str] = properties.attrs or []
-        attrs_string = r"\l".join(attr.replace("|", r"\|") for attr in attrs)
-        label = rf"{{{label}|{attrs_string}\l|"
+        attrs_string = r'<br ALIGN="LEFT"/>'.join(
```
Comment (Pierre-Sassoulas) [inline pylint/pyreverse/dot_printer.py:95]: ``<br ALIGN="LEFT"/>`` could be a class constant instead (and used at line 110 too).

**Review by Pierre-Sassoulas — CHANGES_REQUESTED** [review]
Thank you, it looks pretty good to me, but I'm not a pyreverse expert :)

**Comment by DudeNr33** [comment]
Hi @qequ,
thank you for the contribution!
Sorry for the late response, but my internet at home is down for almost a week now.

Can you also add this feature for the `MermaidJSPrinter`?
For Mermaid, adding a `*` behind the parentheses should do the trick:

```
classDiagram
    class Duck{
      +String beakColor
      +swim()*
      +quack()
    }
```
renders as:
```mermaid
classDiagram
    class Duck{
      +String beakColor
      +swim()*
      +quack()
    }
```

File: pylint/pyreverse/dot_printer.py (hunk 1 above)
Comment (qequ) [reply pylint/pyreverse/dot_printer.py]: Added class constant :+1:

**Comment by qequ** [comment]
@DudeNr33 added italics for mermaidjs :+1:

File: pylint/pyreverse/dot_printer.py [hunk 2]
```diff
@@ -10,13 +10,19 @@
...
 from pathlib import Path
 
 from astroid import nodes
 
 from pylint.pyreverse.printer import EdgeType, Layout, NodeProperties, NodeType, Printer
 from pylint.pyreverse.utils import get_annotation_label
 
+
+class HTMLLabels(Enum):
+    LINEBREAK_LEFT = '<br ALIGN="LEFT"/>'
```
Comment (Pierre-Sassoulas) [inline pylint/pyreverse/dot_printer.py:23]: It's only used in the dot printer so let's keep it in the dot printer as a nested class :)

Comment (qequ) [reply pylint/pyreverse/dot_printer.py:23]: @Pierre-Sassoulas do I have to change the `HTMLLabels` class?

Comment (Pierre-Sassoulas) [reply pylint/pyreverse/dot_printer.py:23]: You can make a single variable instead of an enum too, but both work.

**Review by DudeNr33 — APPROVED** [review]
Thank you for this addition. Looks good to me! 👍

File: pylint/pyreverse/dot_printer.py (hunk 2 above)
Comment (qequ) [reply pylint/pyreverse/dot_printer.py:23]: @Pierre-Sassoulas  I think it's better to have it as a class in case more labels are added in the future

**Review by Pierre-Sassoulas — APPROVED** [review]
Thank you for implementing this feature and congratulation on your first contribution to pyreverse 🎉

**Pull request merged**